packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -133,15 +154,15 @@ export default function LabeledField(props: Props) {
                 <LabelMedium
                     style={[
                         styles.textWordBreak,
-                        light ? styles.lightLabel : styles.label,
+                        isLight ? styles.lightLabel : styles.label,
                     ]}
                     tag="label"
                     htmlFor={fieldId}
                     testId={testId && `${testId}-label`}
                     id={labelId}
                 >
                     {label}
-                    {required && requiredIcon}
+                    {isRequired && requiredIcon}
                 </LabelMedium>
                 <Strut size={spacing.xxxSmall_4} />
```

Comment:
**thought:** Looking at the stories, this got me thinking..... we should probably refactor this at some point to use `gap` instead of `Strut` (as we have been discussing previously). No need to make any changes here, but this would be a really nice improvement in the future (for all WB components).

Reply:
Good catch! I'll update this in a follow up PR :) Thanks!